tests/network/sriov.go

```diff
@@ -689,16 +689,21 @@ func createVMIAndWait(vmi *v1.VirtualMachineInstance) (*v1.VirtualMachineInstanc
 		return nil, err
 	}
 
-	return waitVMI(virtClient, vmi)
+	return waitVMI(vmi)
 }
 
-func waitVMI(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance) (*v1.VirtualMachineInstance, error) {
+func waitVMI(vmi *v1.VirtualMachineInstance) (*v1.VirtualMachineInstance, error) {
 	// Running multi sriov jobs with Kind, DinD is resource extensive, causing DeadlineExceeded transient warning
 	// Kubevirt re-enqueue the request once it happens, so its safe to ignore this warning.
 	// see https://github.com/kubevirt/kubevirt/issues/5027
 	warningsIgnoreList := []string{"unknown error encountered sending command SyncVMI: rpc error: code = DeadlineExceeded desc = context deadline exceeded"}
 	tests.WaitUntilVMIReadyIgnoreSelectedWarnings(vmi, console.LoginToFedora, warningsIgnoreList)
 
+	virtClient, err := kubecli.GetKubevirtClient()
+	if err != nil {
+		panic(err)
```

Comment:
Why panic and not return an err? 
Returning an error may give a chance to other tests to pass instead of stopping the whole test suite.

Reply:
Ginkgo `Fail` also panics, so tests will continue running even with such a direct panic.

You should check the original code and see it also panics here.
The reasoning is that errors in tests better be differentiated from failures in expectations (assertions).